drivers/timer/Kconfig

```diff
@@ -97,6 +97,7 @@ source "drivers/timer/Kconfig.xlnx_psttc"
 source "drivers/timer/Kconfig.xtensa"
 source "drivers/timer/Kconfig.mtk_adsp"
 source "drivers/timer/Kconfig.sy1xx_sys_timer"
+source "drivers/timer/Kconfig.ra8_ulpt"
```

Comment:
Please change file name to easy detect for Renesas RA maintaining.

Reply:
@thaoluonguw , It's fixed